contracts/drafts/Strings.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Strings
+ * @dev String operations.
+ * via https://github.com/oraclize/ethereum-api/blob/master/oraclizeAPI_0.5.sol MIT licence
+ */
+library Strings {
+    /**
+     * @dev Concatenates two strings.
+     */
+    function concatenate(string memory a, string memory b) public pure returns (string memory concatenatedString) {
+        bytes memory bytesA = bytes(a);
```

Comment:
I'm not sure what the semantics of this are, is a new array created? Is it just an alias of the old one? Based on this I have further comments about the rest of the function.